p2p/message.go

```diff
@@ -54,6 +55,40 @@ type Message interface {
 	Id() string
 }
 
+// DecodeMessage accepts a raw message including the type indicator byte and decodes it to its specific message type
+func DecodeMessage(r io.Reader) (m Message, err error) {
```

Comment:
is this used somewhere?

Reply:
It is not. The method that previously lived in this file named `decodeMessage` moved to `p2p/utils.go`. We no longer need this method here or we need to update the method there.

Reply:
sure